src/prognostic_equations/forcing/subsidence.jl

```diff
@@ -1,28 +1,70 @@
 #####
-##### Subsidence forcing
+##### Tendencies due to prescribed subsidence
 #####
 
 import Thermodynamics as TD
 import ClimaCore.Spaces as Spaces
 import ClimaCore.Fields as Fields
 import ClimaCore.Operators as Operators
 
-#####
-##### No subsidence
-#####
+"""
+    subsidence!(ᶜρχₜ, ᶜρ, ᶠu³, ᶜχ, scheme::Val)
 
-subsidence_tendency!(Yₜ, Y, p, t, ::Nothing) = nothing
+Computes the tendency contribution to a density-weighted scalar `ρχ` due to
+subsidence (vertical advection by a prescribed large-scale vertical velocity `ᶠu³`).
+The tendency is calculated as `ρ * (-w * ∂χ/∂z)`, where `w` is `ᶠu³` and `χ` is `ᶜχ`.
```

Comment:
I don't think the second part is accurate, as it's the general coordinate gradient, not ∂ᶜχ/∂z.